src/tests/ftest/osa/online_reintegration.py

```diff
@@ -52,16 +53,22 @@ def run_online_reintegration_test(self, num_pool, racer=False, server_boot=False
                 Defaults to False.
             server_boot (bool) : Perform system stop/start on a rank. Defaults to False.
             oclass (str) : daos object class string (eg: "RP_2G8"). Defaults to None.
+            multiple_ranks (bool) : Perform multiple ranks testing (Default: False)
         """
         if oclass is None:
             oclass = self.ior_cmd.dfs_oclass.value
         test_seq = self.ior_test_sequence[0]
         # Create a pool
         pool = {}
-        exclude_servers = (len(self.hostlist_servers) * 2) - 1
 
-        # Exclude one rank : other than rank 0.
-        rank = self.random.randint(1, exclude_servers)
+        ranklist = list(self.server_managers[0].ranks.keys())
+        if multiple_ranks:
+            rank = self.random.sample(ranklist, k=2)
+            # For multiple rank testing, we need RP_3G1 for IOR to complete.
+            oclass = "RP_3G1"
```

Comment:
This function already has an `oclass` input, so this should be set by the caller when requesting multiple ranks.